ip-allow-listing-route-service/README.md

```diff
@@ -0,0 +1,47 @@
+# IP Allow-List Route Service
+
+A simple GoLang application that implements the [route service contract](https://docs.cloudfoundry.org/services/route-services.html)
+of Cloud Foundry and checks the client IP address against an allow-list before passing on the
+request. This requires the `x-cf-true-client-ip` header to be set by the foundation which is the
+case on [SAP BTP, Cloud Foundry runtime](https://www.sap.com/products/technology-platform/btp-cloud-foundry-runtime.html).
+
+## Usage
+
+> [!CAUTION]
+>
+> This is a sample application that is only intended to demonstrate the general idea. It is not
+> production grade.
+
+To deploy the application you will first need an application you want to put this in front of, as
+well as an allow-list. The allow-list should contain one CIDR per line, empty lines and lines
+starting with a `#` are ignored. Example:
+
+```
+# This is a comment
+10.0.0.0/8
+127.0.0.0/8
+# The following lines will be ignored
+
+
+```
+
+Put this file into the app directory next to the manifest file and name it `allowlist.txt`. Now you
+can push the application like this (you may need to adjust the domain depending on the region):
+
+```
+cf push -f ./manifest.yml --var domain=cfapps.eu10.hana.ondemand.com --var suffix=foo
```

Comment:
I think we use "landscape domain" as term for the other samples, not an explicit SAP landscape URL.

Reply:
I wouldn't want to pin this on a particular landscape. You can also use a made up system domain, e.g. `cf.example.com`?

Reply:
Done.